index.html

```diff
@@ -26,24 +26,181 @@
+                         <div class="project photo"><img src="assets/work/work_five.png" class="work_one" alt="#"></div>
+                         <div class="project summary">
+                            <div class="nameP"><p>Project name goes here</p></div>
+                            <div class="technology">
+                                <div class="tech1"><p>HTML/CSS</p></div>
+                                <div class="tech1"><p>Ruby on Rails</p></div>
+                                <div class="tech1"><p>JavaScript</p></div>
+                            </div>
+                            <div class="project-view">
+                                <div class="word"><p>See this project</p></div>
+                                <div class="union"><img src="assets/work/Union.png" class="arrow" alt="#"></div>
+                            </div>
 
+                         </div>
+                     </div>
+                        
 
+                         <div class="work six">
+                             <div class="project photo"><img src="assets/work/work_six.png" class="work_one" alt="#"></div>
+                             <div class="project summary">
+                                <div class="nameP"><p>Project name goes here</p></div>
+                                <div class="technology">
+                                    <div class="tech1"><p>HTML/CSS</p></div>
+                                    <div class="tech1"><p>Ruby on Rails</p></div>
+                                    <div class="tech1"><p>JavaScript</p></div>
+                                </div>
+                                <div class="project-view">
+                                    <div class="word"><p>See this project</p></div>
+                                    <div class="union"><img src="assets/work/Union.png" class="arrow" alt="#"></div>
+                                </div>
+
+                             </div>
+                             </div>        
+    </section>
+
+    <section class="About">
+        <div class="aboutWord"><h2>About</h2></div>
+        <div class="aboutText"><p>Lorem ipsum 
+            dolor sit amet, consectetur 
+            adipiscing elit, sed do eiusmod 
+            tempor incididunt ut labore et 
+            dolore magna aliqua. Ut enim ad 
+            minim veniam, quis nostrud 
+            exercitation ullamco
+             laboris nisi ut aliquip ea eiusmod</p>
+            </div>
+            <div class="getflex">
+        <div class="cvBox"><p>Get my CV</p>
+        <div class="union_1"><img src="assets/Union_two.png" class="arrow_down" alt="#"></div></div>
+    </div>
+        <div class="gridContainer">
+            <div class="lang"><h4>Languages</h4>
+                <ul>
+                    <li><p>JavaScript</p></li>
+                    <li><p>Ruby</p></li>
+                    <li><p>HTML</p></li>
+                    <li><p>CSS</p></li>
+                </ul>
```

Comment:
This is correct for that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` - Nice job 👍

Reply:
Please, I just made the changes

Reply:
One last thing, you are not supposed to make changes to your repo after you've submitted it for review. Wait for the review comments before making any further changes after submitting your project.